Formula/alpine.rb

```diff
@@ -31,6 +31,9 @@ class Alpine < Formula
     depends_on "linux-pam"
   end
 
+  # patch for macOS obtained from developer; see git commit 701aebc00aff0585ce6c96653714e4ba94834c9c
+  patch :DATA
```

Comment:
Any reason why we're not using the patch at https://repo.or.cz/alpine.git/patch/701aebc00aff0585ce6c96653714e4ba94834c9c?hp=8b55be9f989f88aa0f464ec1c5f7dbef70310055?